.github/workflows/sync_lang_master_with_patch.yml

```diff
@@ -6,7 +6,7 @@ on:
     - cron:  '0 0 * * *'
 jobs:
   send_pr_from_patch_branch:
-    name: Send a nightly PR to master
```

Comment:
Why do you need to remove the name?

Reply:
Since we have a names for each step, this is redundant